packages/replay-next/components/elements/ElementsListData.ts

```diff
@@ -439,6 +452,14 @@ export class ElementsListData extends GenericListData<Item> {
 
     this.updateLoadingState(false);
     this.invalidate();
+
+    if (relativeRootId === this._rootObjectId) {
+      const stopTime = Date.now();
+      recordTelemetryData("elements-panel-load-tree", {
+        duration: stopTime - startTime,
+        params: {},
+      });
+    }
```

Comment:
I don't think the way you have logging setup in this panel makes sense. For example, logging the same event in both `loadAndProcessNodeSubTree` and `loadPathToNode` – when `loadAndProcessNodeSubTree` is called by `loadPathToNode`.

I'd prefer you remove this logging for now. If we want to add logging here, I think I should probably add it (and I'd prefer to add it after we decide about #9904.)